tools/e2e/examples/docker-compose/kuma-example-installer.sh

```diff
@@ -7,7 +7,7 @@ set -e
 #
 
 function resolve_ip {
-  nslookup "${1}" 2>/dev/null | tail -1 | awk '{print $3}'
+  getent hosts ${DATAPLANE_HOSTNAME} 2>/dev/null | awk -e '{ print $1 }'
```

Comment:
That's nice! Do you think we need to get local IP the same way here:
https://github.com/Kong/kuma/blob/master/pkg/plugins/runtime/k8s/webhooks/injector/injector.go#L169
and here: https://github.com/Kong/kuma/blob/master/pkg/plugins/runtime/k8s/webhooks/injector/injector.go#L185

Reply:
Not sure what you mean, but if you refer to resolving `localhost` that might be a thing. You should try it.
BTW all the credit for this specific solution should go to @jakubdyszkiewicz for finding it. I tried other tricks.